jwst/cube_build/cube_build_step.py

```diff
@@ -18,9 +19,8 @@ class CubeBuildStep (Step):
 
     spec = """
          channel = option('1','2','3','4','ALL','all',default='ALL') # Options: 1,2,3,4,or All
```

Comment:
Is there a reason to list all capitalizations of each spec?  Shouldn't it just allow a single one and make it consistent?  I.e. lowercase?

That said, I notice that our current error message is not very good in helping users. I.e. if a wrong option is specified, we currently do not let the user know what the correct option is.  That should be improved I think.  I will file an issue.
```
In [1]: from jwst.cube_build import CubeBuildStep

In [2]: step = CubeBuildStep(band='foo')
---------------------------------------------------------------------------
ValidationError                           Traceback (most recent call last)
<ipython-input-3-39016978c84c> in <module>()
----> 1 step = CubeBuildStep(band='foo')

~/dev/jwst/jwst/stpipe/step.py in __init__(self, name, parent, config_file, _validate_kwds, **kws)
    286             spec = self.load_spec_file()
    287             kws = config_parser.config_from_dict(
--> 288                 kws, spec, root_dir=dirname(config_file or ''))
    289 
    290         if name is None:

~/dev/jwst/jwst/stpipe/config_parser.py in config_from_dict(d, spec, root_dir)
    194 
    195     if spec:
--> 196         validate(config, spec, root_dir=root_dir)
    197     else:
    198         config.walk(string_to_python_type)

~/dev/jwst/jwst/stpipe/config_parser.py in validate(config, spec, section, validator, root_dir)
    255 
    256         if len(messages):
--> 257             raise ValidationError('\n'.join(messages))
    258     finally:
    259         config.main.configspec = orig_configspec

ValidationError: Config parameter 'band': the value "foo" is unacceptable.
```